core/design/src/main/kotlin/edu/stanford/spezi/core/design/views/views/layout/DescriptionGridRow.kt

```diff
@@ -0,0 +1,75 @@
+package edu.stanford.spezi.core.design.views.views.layout
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.HorizontalDivider
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import edu.stanford.spezi.core.design.theme.ThemePreviews
+
+@Composable
+fun DescriptionGridRow(
+    description: @Composable () -> Unit,
+    modifier: Modifier = Modifier,
+    content: @Composable () -> Unit,
+) {
+    Row(
+        modifier = modifier
+            .fillMaxWidth()
+            .padding(vertical = 8.dp),
```

Comment:
We have already some predefined constants, please use `Spacings.small` here and below instead

Reply:
trust me it wont create any issue, it is part of a design system and it serves for consistency throughout the project instead of having spreaded magic numbers. If needed, we can even make the values configurable so that they can be passed on the spezi theme directly. Also, the values that we are using there are pretty standard ones common for every design system so we should be good to go